automobile/sumo-interface.md

```diff
@@ -30,25 +35,31 @@ field `controllerArgs` in order to customize the behavior of the interface:
+| --verbose                | Prints SUMO outputs in Webots console                                                                                                | SUMO outputs are lost                                                                            |
+| --step                   | Specifies the time step of SUMO in milliseconds                                                                                      | By default, a time step of 200ms is used                                                         |
+| --no-netconvert          | Does not run netconvert before launching SUMO                                                                                        | By default, netconvert is run before launching SUMO. It is required to disable netconvert if the *.net.xml file is already available (for example when importing it from OpenStreetMap)          |
+| --disable-traffic-lights | Does not update traffic light states in Webots                                                                                       | By default, traffic light states are updated in Webots                                           |
+| --max-vehicles           | Specifies the maximum number of vehicles included in Webots                                                                          | By default, a maximum of 100 vehicles are included in Webots                                     |
+| --rotate-wheels          | Updates the orientation of the wheels of the vehicles in Webots                                                                      | By default, the wheels orientation is not updated                                                |
+| --radius                 | Specifies the visibility radius of the vehicles in meters (only vehicles closer to the viewpoint than this radius will be displayed) | By default, the visibility radius is equal to -1 which means that all the vehicles are displayed |
+| --enable-height          | Specifies if height information should be extracted from the edge name                                                               | By default, all the vehicles are in the plane defined by y = 0                                   |
+| --d or --directory       | Specifies the directory where are located the SUMO network files                                                                     | By default the files should be located close to the world in a directory called `worldName_net`  |
+| --p --port               | Specifies which port SUMO and the interface should use to communicate                                                                | By default, the port 8873 is used                                                                |
+| --s --seed               | Specifies the seed of the SUMO random number generator (0 for the '--random' option of SUMO)                                         | By default, a seed of 1 is used                                                                  |
```

Comment:
I would remove the short versions of these options, e.g., `--d`, `--p` and `--s`.